packages/ui/src/machines/authenticator/defaultServices.ts

```diff
@@ -14,13 +14,26 @@ export const defaultServices = {
   // Validation hooks for overriding
   async validateCustomSignUp(formData): Promise<ValidatorResult> {},
   async validateConfirmPassword<Validator>(formData): Promise<ValidatorResult> {
-    const { password, confirm_password } = formData;
+    const {
+      password,
+      confirm_password,
+      touched_confirm_password,
+      touched_password,
```

Comment:
Yeah that would make sense. Perhaps, we could add this in a future release? Seems like a big task

Reply:
What if I create a new `formControls` context, that will have all the `touched.*` values, at least for right now just for `password` and `confirm_password`. We can build on this to add in more form validation for the future, and it will remove the values out of `formValues` into their own separate object.  I'll push that up, so I can get your feedback.

Reply:
Pushed in 032b48b

Reply:
Awesome! One last nit -- what do you think about renaming `formControls` to just `touched` for now, and let the keys be `${name}` instead of `touched_${name}`? Then when we need other `formControls`, we can expose them as separate facade. I think this separation of form facades will improve DX than having a general `formControls`, on the long run. What do you think @ericclemmons @ErikCH?

This will need some modification to `runValidators` shape though... 😓  but I think now is a good time to make changes, before we document them!

Reply:
Sure! I updated the name to `touched` and update the validators 15c8b02 and 86533e8